### PR TITLE
350 add doc entry about multi-user

### PR DESCRIPTION
**Pull request by leite08**
Ref: metriport/metriport-internal#350

### Dependencies

- Upstream: https://github.com/metriport/metriport-internal/pull/1331
- Downstream: none

### Description

Add doc entry about multi-user, called "Invite your team"

image

image

### Testing

- Local
  - [x] Tested the page local, including newly added links
- Staging
  - no docs on staging
- Production
  - [ ] Validate it displays properly

### Release Plan

- [ ] Added to monthly product update [here](https://www.notion.so/metriport/December-2023-7945176d59344e72a7c0e39f43c8d53f?pvs=4#e54a193b523040efa158cef4546b0bb8)
- [ ] Upstream dependencies are met
- [ ] Merge this

### CODE REVIEW

File: docs/medical-api/getting-started/quickstart.mdx [hunk 1]
```diff
@@ -254,6 +255,17 @@ Metriport makes it easy to share this data back with the networks - you can:
 - upload FHIR data using our [Create Patient's Consolidated Data](/medical-api/api-reference/fhir/create-patient-consolidated) endpoint.
 - upload binary documents using the [Upload Document](/medical-api/api-reference/document/post-upload-url) endpoint.
 
+## 10. Invite your team 🤝
+
+You can invite your team members to signup at https://dash.metriport.com/ and send us a list of the emails to be associated with your main account.
```
Comment (RamilGaripov) [inline docs/medical-api/getting-started/quickstart.mdx:260]: perhaps clarify where they can send us the list of emails

**Review by RamilGaripov — APPROVED** [review]
LGTM. Added one comment for consideration

File: docs/medical-api/getting-started/quickstart.mdx (hunk 1 above)
Comment (leite08) [reply docs/medical-api/getting-started/quickstart.mdx:260]: That's a good point, but currently the channels used for communication w/ our customers are pretty clear, they use either Slack or email.

I'd leave for a new PR once we learn more how this process will work.

**Review by Orta21 — APPROVED** [review]

**Comment by leite08** [comment]
Closed as this will be part of customer onboarding, not for getting started/sandbox

**Pull request closed without merging**